### DISCUSSION TITLE
Trice not showing colours on v0.53.0

### DISCUSSION THREAD

**Discussion by CMLoureiro** [rokath/trice#298] [Q&A] [answered]
Hello Thomas,

I noticed that on release v0.53.0 Trice is not showing any colours and the traced file name and trace line numbers appear between the time stamp and the labels ATT, WRN, etc
Please see some example trace lines:

1697741957         **consumer_0.c 305**ATT: Acquire: Total[42],Available[4]
1697742191  **producer_0.c 237**MSG:[(0)--»] T[1]::E[3]::M[16758]-SWITCH[2] OFF
1697742194  **producer_0.c 250**WRN:(P0)((POST)): Subscriber not found.

I generate the til.jon file with "trice u" command and start the trace with "trice log -port /dev/ttyUSB1 -baud 1497600"
The host OS is Ubuntu.

Do I require any different option for v0.53.0 when compared to v0.52.0 ?

Thank you for your time,
Carlos

**Comment by rokath** [comment]
Are you using the pre-build binary or did you check out the main branch and compiled? I am changing the location information right now but it should not have affected the binaries.

**Reply by CMLoureiro** [reply]
Hi Thomas,
I checked out tag v0.53.0 and compiled. Followed the same procedure as for  v0.52.0.

**Reply by rokath** [reply]
Would you please tryout the prebuild binary, @CMLoureiro, and tell me the result? Currentlly I am very short of time. Hopefully I can deal with it soon.

**Reply by CMLoureiro** [reply]
Hello Thomas,
I've tried with prebuilt binary:

cmjl:~/go/bin$ trice ver
version=0.53.0, commit=8cb1b1dc28da074f7bef5b7be4387a8380d405cd, built at 2022-06-14T16:34:38Z

but still have the same issue.
Then I tried to run a locally built v0.52.0 with the "til.json" generated from the pre-built v0.53.0 and didn't observe the issue. 

Please take your time. No worries

**Comment by rokath** [comment, marked as answer]
Would you please check v0.54.0? I shows colors also on my linux (Antix).

**Reply by CMLoureiro** [reply]
Hi Thomas,
It's working fine on v0.54.0. Thank you very much!
Antix.. a distro I didn't know :-)

Just one last note. I got a failure when running the "go test ./..." command:

_cmjl:~/projects/trace/trice ((v0.54.0))$ go test ./...
?       github.com/rokath/trice/cmd/cui [no test files]
ok      github.com/rokath/trice/cmd/trice       0.002s
ok      github.com/rokath/trice/internal/args   0.003s
ok      github.com/rokath/trice/internal/com    0.062s
ok      github.com/rokath/trice/internal/decoder        0.002s
ok      github.com/rokath/trice/internal/emitter        0.003s
ok      github.com/rokath/trice/internal/id     0.004s
ok      github.com/rokath/trice/internal/keybcmd        0.002s
ok      github.com/rokath/trice/internal/link   0.001s
ok      github.com/rokath/trice/internal/receiver       0.002s
ok      github.com/rokath/trice/pkg/cipher      0.002s
?       github.com/rokath/trice/pkg/cobs        [no test files]
ok      github.com/rokath/trice/pkg/endian      0.001s
ok      github.com/rokath/trice/pkg/msg 0.001s
i=0, exp := []byte{ 0x02, 0x01, 0x01, 0x01, 0x0d, 0x11, 0x11, 0x11, 0x11, 0xc0, 0x01, 0x83, 0xe5, 0xff, 0xff, 0xff, 0xff, 0x00, 0x00, 0x00, }
--- FAIL: TestTriceSequences (0.00s)
    require.go:72: 
                Error Trace:    require.go:72
                                                        trice_test.go:41
                Error:          Not equal: 
                                expected: []byte{0x2, 0x3, 0x1, 0x1, 0x2, 0x15, 0xf, 0x38, 0xcb, 0x11, 0x11, 0x11, 0x11, 0xc0, 0x1, 0x83, 0xe5, 0xff, 0xff, 0xff, 0xff, 0x0, 0x0, 0x0}
                                actual  : []byte{0x2, 0x1, 0x1, 0x1, 0xd, 0x11, 0x11, 0x11, 0x11, 0xc0, 0x1, 0x83, 0xe5, 0xff, 0xff, 0xff, 0xff, 0x0, 0x0, 0x0}
                            
                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1,4 +1,4 @@
                                -([]uint8) (len=24) {
                                - 00000000  02 03 01 01 02 15 0f 38  cb 11 11 11 11 c0 01 83  |.......8........|
                                - 00000010  e5 ff ff ff ff 00 00 00                           |........|
                                +([]uint8) (len=20) {
                                + 00000000  02 01 01 01 0d 11 11 11  11 c0 01 83 e5 ff ff ff  |................|
                                + 00000010  ff 00 00 00                                       |....|
                                 }
                Test:           TestTriceSequences
FAIL
FAIL    github.com/rokath/trice/pkg/src 0.002s
ok      github.com/rokath/trice/pkg/tst 0.002s
FAIL_

**Comment by th-sabik** [comment]
Thanks for the hint. Fixed in v0.54.1. I forgot to adapt testdata. They do not contain binary location information data anymore.
